### ISSUE TITLE
Weird added spaces after adding link

### ISSUE THREAD

**Issue by taxaos** [jpuri/react-draft-wysiwyg#676]
Hello,

thanks for your wonderful app, we like it a lot.

However, we noticed a small but weird bug with adding links.

Following feature: Link.addLink

`// insert a blank space after link
    selection = newEditorState.getSelection().merge({
      anchorOffset: selection.get('anchorOffset') + linkTitle.length,
      focusOffset: selection.get('anchorOffset') + linkTitle.length,
    });`

1. Mark word in text with mouse doubleclick - everything works, space is added after word. OK
2. Mark word with keyboard from left to right - everything works, space is added after word. OK
3. Mark word with keyboard from right to left - now the space gets added xx chars later (xx is the length of linkTitle.length) - NOK

Basically, the selection does not work properly, if you mark the word with keyboard from right to left.

Bug in React-Draft-Wysiwyg or DraftJS itself?

**Comment by Guillaumez** [comment]
Same issue when adding two links :
- Add raw text like this : `Link 1 | Link 2`
- Select `Link 1` and add a random link
- `Link 2` becomes `Lin k 2`. An extra space has been added.

**Comment by pratiknikam** [comment]
I'm having same issue. In my case, it also removed space before link so basically concatenates linked word with previous non-linked word which makes it look worse.

**Comment by chiomadans1759** [comment]
@Guillaumez @pratiknikam @taxaos Do you have an idea if this bug has been fixed yet? I am still having this bug till now
